src/thunderbots/software/ai/hl/stp/play/play.h

```diff
@@ -111,11 +111,10 @@ class Play
      *
      * @param yield The coroutine push_type for the Play
      *
-     * @return A list of shared_ptrs to the Tactics the Play wants to run at this time, in
+     * @yield A list of shared_ptrs to the Tactics the Play wants to run at this time, in
      * order of priority
```

Comment:
`@yield` is non-standard javadoc, we should instead move this into the block comment for this function, and explain that instead of returning values they are "yielded" up by calling `yield`.

Reply:
fixed